pom.xml

```diff
@@ -84,7 +84,7 @@
 	</dependencies>
 
 	<properties>
-		<openMRSVersion>1.9.2</openMRSVersion>
+		<openMRSVersion>1.10.2</openMRSVersion>
```

Comment:
It's fine.  You can only choose one version.  I typically choose the lowest supported version.  But an equally (or more) valid approach could be the most widely used version.